File: 231226/코로나 메뉴얼/covid-manual.py
per1 = input().split()
c1, t1 = per1[0], int(per1[1])

per2 = input().split()
c2, t2 = per2[0], int(per2[1])

per3 = input().split()
c3, t3 = per3[0], int(per3[1])

if c1 == 'Y' and t1 >= 37:
    if (c2 == 'Y' and t2 >= 37) or (c3 == 'Y' and t3 >= 37):
        print("E")
    else:
        print("N")
else:
    if (c2 == 'Y' and t2 >= 37) and (c3 == 'Y' and t3 >= 37):
        print("E")
    else:
        print("N")

File: 240201/코로나 메뉴얼/covid-manual.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split("\n");

let count = 0;
for (let i = 0; i < 3; i++) {
    let p = input[i].split(" ");
    if (p[0] == 'Y' && Number(p[1]) >= 37) {
        count += 1;
    }
}

if (count >= 2) {
    console.log("E");
} else {
    console.log("N");
}